### PR TITLE
Add persona cli

### PR DESCRIPTION
**Pull request by kidkcl**
#341

### CODE REVIEW

File: src/cli/providers/messenger/persona.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import { bold, error, print } from '../../shared/log';
+
+const help = () => {
+  console.log(`
+  bottender messenger persona <command> [option]
+
+  ${chalk.dim('Commands:')}
+
+    list              List all personas.
+    create            Create new personas.
```
Comment (chentsulin) [inline src/cli/providers/messenger/persona.js]: Create a new persona

Comment (kidkcl) [reply src/cli/providers/messenger/persona.js]: The create command should create only one persona like
`persona create <NAME> <PROFILE_IMAGE_URL>`
or, given an array of personas in config, create by updated different ones like set messenger profile did?

File: src/cli/providers/messenger/persona.js [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+    list              List all personas.
+    create            Create a new persona with name and profile picture url.
+    get               Get persona by persona ID.
+    del, delete       Delete persona by persona ID.
+
+  ${chalk.dim('Options:')}
+
+    -t, --token       Specify Messenger access token.
+    --name            Specify persona's name when create
+    --url             Specify persona's profile image url when create
```
Comment (tw0517tw) [inline src/cli/providers/messenger/persona.js]: How about `--image`?

File: src/cli/providers/messenger/persona.js [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+    --id              Specify persona's ID to get or delete
+
+  ${chalk.dim('Examples:')}
+
+  ${chalk.dim('-')} Create a new persona
+
+    ${chalk.cyan(
+      '$ bottender messenger persona create --name <PERSONA_NAME> --url <PROFILE_PIC_URL>'
```
Comment (tw0517tw) [inline src/cli/providers/messenger/persona.js]: `<PROFILE_IMAGE_URL>` is better align with the help text.

File: src/cli/providers/messenger/persona.js [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+    )}
+
+  ${chalk.dim('-')} Get persona by ID
+
+    ${chalk.cyan('$ bottender messenger persona get --id <PERSONA_ID>')}
+
+  ${chalk.dim('-')} Delete persona with specific access token
+
+    ${chalk.cyan(
+      '$ bottender messenger persona delete --token __FAKE_TOKEN__ --id <PERSONA_ID>'
```
Comment (tw0517tw) [inline src/cli/providers/messenger/persona.js]: replace `__FAKE_TOKEN__` with `<ACCESS_TOKEN>`

File: src/cli/providers/messenger/persona.js (hunk 2 above)
Comment (tw0517tw) [reply src/cli/providers/messenger/persona.js]: or `--pic`

File: src/cli/providers/messenger/persona.js (hunk 3 above)
Comment (tw0517tw) [reply src/cli/providers/messenger/persona.js]: or change help text to
> Specify persona's profile picture url when create

File: src/cli/providers/messenger/persona.js [hunk 5]
```diff
@@ -0,0 +1,242 @@
...
+      break;
+    case 'get':
+      await getPersona(ctx);
+      break;
+    case 'delete':
+    case 'del':
+      await deletePersona(ctx);
+      break;
+    default:
+      error(`Please specify a valid subcommand: set`);
```
Comment (tw0517tw) [inline src/cli/providers/messenger/persona.js]: all subcommands should be listed here

Comment (chentsulin) [inline src/cli/providers/messenger/persona.js]: create, list, get, delete

File: src/cli/providers/messenger/persona.js [hunk 6]
```diff
@@ -0,0 +1,242 @@
...
+    }
+    return process.exit(1);
+  }
+}
+
+export default async function main(ctx) {
+  const subcommand = ctx.argv._[2];
+  switch (subcommand) {
+    case 'create':
+      await createPersona(ctx);
```
Comment (Calvin-Lin) [inline src/cli/providers/messenger/persona.js]: 這邊其實可以直接 return

File: src/cli/providers/messenger/persona.js [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+  ${chalk.dim('-')} Delete persona with specific access token
+
+    ${chalk.cyan(
+      '$ bottender messenger persona delete --token <ACCESS_TOKEN> --id <PERSONA_ID>'
+    )}
+`);
+};
+
+export async function createPersona(ctx) {
+  const { t, token: _token, name: personaName, pic: personaUrl } = ctx.argv;
```
Comment (kpman) [inline src/cli/providers/messenger/persona.js]: why rename token to `_token`?

**Pull request merged**